example-docs/language-docs/eng_spa_mult.eml

```diff
@@ -79,15 +79,15 @@ re endowed with reason and conscience and should act towards one another in=
 ty and rights. They are endowed with reason and conscience and should act t=
 owards one another in a spirit of brotherhood. All human beings are born fr=
 ee and equal in dignity and rights. They are endowed with reason and consci=
-ence and should act towards one another in a spirit of brotherhood.<br><br>=
+ence and should act towards one another in a spirit of brotherhood. <p>    =

```

Comment:
Was the example doc here supposed to change?

Reply:
This email is used by the test that verifies the `detect_language_per_element` behaviors for EML. That test depends on the body of this email resolving to distinct paragraphs so it can verify each Element (paragraph) was assigned a language separately.

The original HTML text of this email is not separate paragraphs, it just "looks" like separate paragraphs. So that test started failing.

It's a legitimate question whether we want to "double-parse" `<div>` elements like this one and consider `<br/><br/>` to be a paragraph boundary. But that's not the behavior this email's test is verifying so I went with the element change.